cmd/renterd/main.go

```diff
@@ -571,6 +566,25 @@ func main() {
 
 	// Start server.
 	go srv.Serve(l)
+
+	// Set initial S3 keys.
+	if cfg.S3.Enabled && !cfg.S3.DisableAuth {
```

Comment:
`nit` too late to get rid of double neg? I've been biting my tongue and don't really feel strongly about it but I'm sure you noticed it was a bit weird when writing this condition right. Change to `authEnabled` and deafult to true?

Reply:
I agree it's not the prettiest in code but imo for someone who is using the software I feel the negative is better. Disable makes it seem like setting this is something you usually don't want to do imo.